velox/functions/prestosql/tests/FunctionBaseTest.h

```diff
@@ -368,6 +368,25 @@ class FunctionBaseTest : public testing::Test {
     return vectorMaker_.arrayVector<T>(size, sizeAt, valueAt, isNullAt);
   }
 
+  // Convenience function to create vector from a base vector.
```

Comment:
this comment is probably better to be inside the function in the vector maker

Reply:
There is a similar comment there